rbac/ledger_sync/subscriber.py

```diff
@@ -73,7 +73,6 @@ def start(self, known_ids=None):
                 ),
             ],
         )
-
```

Comment:
There's a couple of times in this PR where you just deleted lines from the files. Because you didn't actually remove or add code, can you add the lines back?

Reply:
re-added the earlier empty lines